go-controller/pkg/retry/obj_retry.go

```diff
@@ -93,7 +92,6 @@ func NewRetryFramework(
 		retryEntries:    syncmap.NewSyncMap[*retryObjEntry](),
 		retryChan:       make(chan struct{}, 1),
 		watchFactory:    watchFactory,
-		StopChan:        stopChan,
```

Comment:
Great! Don't forget to also remove it from `pkg/ovn/obj_retry_master.go`
https://github.com/ovn-org/ovn-kubernetes/blob/c72e852628872cd75d1fb5db26fc758ee7d91bdc/go-controller/pkg/ovn/obj_retry_master.go#L61-L65

Reply:
ack

Reply:
I dont think we can remove the stopchannel here, as its still part of the retry framework, we just dont need to use it in the periodic retry, but it is used elsewhere

Reply:
I dont think we can remove the controller stopchannel here, we still have to pass that to the retry framework, we just dont need to use it in periodic retry

Reply:
That's the only place in the code where we use it :)

Reply:
In short, you've changed the signature of `NewRetryFramework`, which we call in obj_retry_master.go, so now when we call NewRetryFramework we won't pass `oc.stopChan` anymore.

Reply:
https://github.com/ovn-org/ovn-kubernetes/blob/c72e852628872cd75d1fb5db26fc758ee7d91bdc/go-controller/pkg/retry/obj_retry.go#L89

The retry framework is expecting a stopChannel to be passed in, if we can remove this as well then we can remove it from master retry

Reply:
That was the only place in the code where we used that stopChannel, so i removed it from the func definition as well as from master retry